### ISSUE TITLE
Can't Patch Firmware

### ISSUE THREAD

**Issue by Polar0fficial** [flipperdevices/flipperzero-firmware#3047]
### Describe the bug.

In regards to this blog post, "https://techryptic.github.io/2023/09/01/Annoying-Apple-Fans/" it tells me to use this command to patch the firmware "./fbt COMPACT=1 DEBUG=0 VERBOSE=1 updater_package" The part I'm confused about is where to execute that command.

### Reproduction

No idea where to execute this command. If anyone can help, it would be greatly appreciated.

**Comment by AdminEvilDRagon** [comment]
I have the same problem/issue

**Comment by AdminEvilDRagon** [comment]
I found the problem you need to remove the ./ 
like this

fbt COMPACT=1 DEBUG=0 VERBOSE=1 updater_package

**Comment by Polar0fficial** [comment]
I tried that, but I got the following error. "fbt : The term 'fbt' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the
spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ fbt COMPACT=1 DEBUG=0 VERBOSE=1 updater_package
+ ~~~
    + CategoryInfo          : ObjectNotFound: (fbt:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException"

I'm supposed to type this in the command prompt application on my PC, right?
